.github/workflows/docker-build.yml

```
@@ -0,0 +1,50 @@
+  push:
+    branches: [ main ]
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Pass SHA and Is_Release to Dockerfile Args
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:29: Double quote to prevent globbing and word splitting

(shellcheck)